Example/Source/Examples/Interactive/InteractiveAlignViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  InteractiveAlignViewController.swift
+//  BrickKit-Example
+//
+//  Created by Ruben Cagnie on 12/9/16.
+//  Copyright © 2016 Wayfair LLC. All rights reserved.
+//
+
+import UIKit
+import BrickKit
+
+class ScaleInsert: BrickAppearBehavior {
```

Comment:
Should this class be named `ScaleInsertBehavior` so that the name signifies what it is?